README.md

```diff
@@ -15,7 +15,7 @@ Prerequisites: JDK 11, update Intellij to the most recent version.
    1. Click `Open or Import`.
    1. Select the project directory, and click `OK`
    1. If there are any further prompts, accept the defaults.
-1. After the importing is complete, locate the `src/main/java/Duke.java` file, right-click it, and choose `Run Duke.main()`. If the setup is correct, you should see something like the below:
+1. After the importing is complete, locate the `src/main/java/duke.duke.Duke.java` file, right-click it, and choose `Run duke.duke.Duke.main()`. If the setup is correct, you should see something like the below:
```

Comment:
Is it entirely necessary to have a duke package within another duke package? 🤔

Reply:
You're right! I think I'll rename it since it now houses the gui components.